### PR TITLE
Do not use dev/generator to build integrations

### PR DESCRIPTION
**Pull request by mtojek**
Issue: https://github.com/elastic/package-registry/issues/466#issuecomment-642022815

This is a temporary solution to remove package build logic from the `package-registry`. Nearly all of these targets will go to the dedicated build tool.

Changes:
* find all packages (package-storage and integrations)
* copy package source to the destination (rsync)
* verify all required fields
* encode Kibana dashboards in build directory

### CODE REVIEW

File: magefile.go [hunk 1]
```diff
@@ -7,15 +7,25 @@
...
 
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
+	"github.com/pkg/errors"
+
+	"github.com/elastic/package-registry/util"
 )
 
+const packageRegistryConfigYML = `public_dir: "./public"
+cache_time.search: 10m
```
Comment (ruflin) [inline magefile.go:25]: Does it need all configs or can it just be an empty file and takes the defaults? We should probably make it possible that it also runs if no config files is there it not already the case.

Comment (mtojek) [reply magefile.go:25]: > We should probably make it possible that it also runs if no config files is there it not already the case.

We need a fix in the registry then.

Comment (ruflin) [reply magefile.go:25]: Ok, lets get it in and follow up.

Comment (mtojek) [reply magefile.go:25]: I still need to update the dependency on changes in https://github.com/elastic/package-registry/pull/507 .

Comment (mtojek) [reply magefile.go:25]: > Ok, lets get it in and follow up.

https://github.com/elastic/package-registry/pull/508

Comment (mtojek) [reply magefile.go]: Fixed

**Pull request merged**